docs/migrating-from-0.9.md

```diff
@@ -0,0 +1,251 @@
+# Migrating from 0.9
+
+Genkit 1.0 introduces some of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.9, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
+If you're using 0.5, use <> to upgrade to 0.9 first before upgrading to 1.0
+
+
+## Streaming functions do not require an await
+
+
+**Old:**
+
+```ts
+const { stream, response } = await ai.generateStream(`hi`);
+const { stream, output } = await myflow.stream(`hi`);
+```
+
+**New**
+
+```ts
+const { stream, response } = ai.generateStream(`hi`);
+const { stream, output } = myflow.stream(`hi`);
+```
+
+
+## Beta APIs
+
+We're introducing an unstable, Beta API channel, and leaving session, chat and Genkit client APIs in beta as we continue to refine them. 
```

Comment:
I think we need to be explicit here about `createSession` and `chat` as the nouns, so people can control-f in this doc and find the fix for any errors

Reply:
Done.